doc/classes/String.xml

```diff
@@ -760,6 +760,7 @@
 			<param index="0" name="file" type="String" />
```

Comment:
It's probably better to rename this argument to `path`. It should not be a breaking change, since number and type of arguments is not changed.